Assets/StreamingAssets/Quests/A0C00Y16.txt

```diff
@@ -217,11 +191,7 @@ Foe _F.00_ is 3 Barbarian
 	start timer _S.16_ 
 	log 1010 step 0 
 	pick one of _S.00_ _S.02_ _S.03_ _S.01_ 
-	create npc _friend4_ 
-	create npc _friend2_ 
-	create npc _friend3_ 
-	create npc _friend1_ 
-	
```

Comment:
I don't agree with this change either, if you remove that, NPCs won't be placed at all. Or maybe this is not needed in DFU, I need to check the code, but don't have much time to do that at the moment. At least in classic, I'm certain that this is required to place NPCs.

Reply:
I'm not familiar with the new changes to the quest engine, and don't believe this PR will accomplish anything good at this point. I'd rather leave it alone rather than makes things worse.

I'm quite certain create NPC is unnecessary in DFU (it was required in classic but NPCs are automatically generated when the quest is started in DFU). Other than that, I'd likely cause problems with this. I'll withdraw it.